setup.py

```diff
@@ -92,7 +93,11 @@
         ],
         'invenio_jsonschemas.schemas': [
             'records_files = invenio_records_files.jsonschemas',
-        ]
+        ],
+        'invenio_base.api_blueprints': [
+            'invenio_records_files = invenio_records_files.'
+            'views:records_files_blueprint',
```

Comment:
This should be [`create_blueprint_from_app`](https://github.com/inveniosoftware/invenio-records-files/pull/79/files#diff-710bd81991051aec95e12970657638f8R20)